index.d.ts

```diff
@@ -45,6 +45,8 @@ declare namespace mapObject {
 		*/
 		shouldRecurse?: boolean;
 	}
+
+	export type mapObjSkip = symbol
```

Comment:
This is not correct. I would recommend reading up on `unique symbol` in TypeScript.

Reply:
I saw it implemented at https://github.com/sindresorhus/p-map/blob/main/index.d.ts#L93 and I found the docs for it at https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#unique-symbol. 

But this is not an ES Module yet, so we can't just do this, right?

```ts
export const mapObjectSkip: unique symbol;
```

I think we need to define the `mapObjectSkip` on the `mapObject` namespace, and I tried several approaches, but couldn't get it to work.

How would you do it?

Reply:
Just use `declare const` at the top-level to declare the `unique symbol` and then add it to the namespace.

Reply:
done via 1efe1a5